### PR TITLE
Feat(UI)/SRT-73: Keep UI feedback info-page missing image

### PR DESCRIPTION
**Pull request by astanovferuz**
Image is set and the section responsiveness is fixed.

### CODE REVIEW

**Review by micahjlucas — APPROVED** [review]
@astanovferuz Looks good on my side 💥 

Please allow @FrozenIce0617 to review and feel free to merge after.

File: src/css/info.scss [hunk 1]
```diff
@@ -39,37 +45,40 @@
...
+          margin-left: 15%;
+          height: 400px;
+          opacity: 0.05;
+      }
+    };
+
+    .privacyColumn  {
+      align-self: center;
+
+      @media (max-width: #{$bp-large}) {
```
Comment (FrozenIce0617) [inline src/css/info.scss:72]: @astanovferuz, As I mentioned yesterday, for the `max-width`, it should smaller `$bp-large - 1`.

File: src/css/info.scss [hunk 2]
```diff
@@ -39,37 +45,40 @@
...
+        padding-left: 45px;
+        h2  {
+          font-size: 45px;
```
Comment (FrozenIce0617) [inline src/css/info.scss:75]: `font-size` and `line-height` are already defined in global. Don't need to redefine it in here.

File: src/css/info.scss [hunk 3]
```diff
@@ -39,37 +45,40 @@
...
+          line-height: 50px;
+        }
+        p {
+          font-size: 18px;
```
Comment (FrozenIce0617) [inline src/css/info.scss:79]: `font-size` and `line-height` are already defined in global. Don't need to redefine it in here.

**Review by FrozenIce0617 — CHANGES_REQUESTED** [review]

**Review by FrozenIce0617 — APPROVED** [review]
LGTM!

**Review by micahjlucas — APPROVED** [review]

**Pull request merged**